node/cn/tracers/api.go

```diff
@@ -57,11 +59,115 @@ const (
 	// trace.
 	defaultTraceReexec = uint64(128)
 
+	// defaultTracechainMemLimit is the size of the triedb, at which traceChain
+	// switches over and tries to use a disk-backed database instead of building
+	// on top of memory.
+	// For non-archive nodes, this limit _will_ be overblown, as disk-backed tries
+	// will only be found every ~15K blocks or so.
+	defaultTracechainMemLimit = common.StorageSize(500 * 1024 * 1024)
```

Comment:
Isn't it too small for tracing? Since our EN node spec is much higher than Ethereum, how about increasing the memory limit?

Reply:
I'm not sure this is not sufficient, but i agree with that our EN node can provide more memory because memory spec is four times larger than ethereum memory spec (ethereum->16GB+, klaytn->64GB+). Reflected at https://github.com/klaytn/klaytn/pull/1581/commits/397e334485b0c0bf5065cd6f63063e4d3313a0d4